### ISSUE TITLE
Besu/Nimbus post merge error: "Engine API reporting different terminal block hash"

### ISSUE THREAD

**Issue by ibhagwan** [status-im/nim-beacon-chain#4122]
**Describe the bug**

Since the merge I'm seeing the below errros on the Nimbus side:
```
WRN 2022-09-15 03:26:00.813-04:00 Engine API reporting different terminal block number topics="eth1" engineAPI_value=0 localValue=15537393
WRN 2022-09-15 03:26:00.814-04:00 Engine API reporting different terminal block hash topics="eth1" engineAPI_value=0x0000000000000000000000000000000000000000000000000000000000000000 localValue=0x55b11b918355b1ef9c5db810302ebad0bf2544255b530cdce90674d5887bb286
```

On the Besu side I see:
```
2022-09-15 03:26:00.811-04:00 | vert.x-worker-thread-0 | WARN  | EngineExchangeTransitionConfiguration | Configured terminal block hash 0x0000000000000000000000000000000000000000000000000000000000000000 does not match value of consensus client 0x55b11b918355b1ef9c5db810302ebad0bf2544255b530cdce90674d5887bb286
```

Using latest nimbus client:
```
❯ nimbus_beacon_node --version
Nimbus beacon node v22.9.0-667c3c-stateofus
Copyright (c) 2019-2022 Status Research & Development GmbH

eth2 specification v1.2.0-rc.3

Nim Compiler Version 1.2.16 [Linux: amd64]
```

Besu:
```
❯ besu --version
besu/v22.7.2/linux-x86_64/openjdk-java-17
2022-09-15 03:28:11.680-04:00 | main | INFO  | Besu | Using jemalloc
```

Acorrding to Adrian Sutton on the Besu discord: "I suspect that's a nimbus bug. The terminal block hash configured should be 0. I think nimbus is reporting what it wound up being instead of what the configured override was"

Seeing other users on discord reporting the same issue with similar block hash values as posted above.

**Comment by hukkin** [comment]
I'm also having this with Nethermind/Nimbus.

**Comment by ZaneCEO** [comment]
I'm not validating myself, but I'm getting multiple reports of this same issue from my customers. Besu+Nimbus

**Comment by ibhagwan** [comment]
Seems to be some sort of miscomunication between the EL and CL layers, (gathered from the corresponding discord channels) the EL devs expected the CL to report 0 and the CL devs expected the EL to report the actual TTD, both are saying this is just noise but and the warning can be safely ignored (on both sides).

**Comment by ibhagwan** [comment]
Source: https://discord.com/channels/613988663034118151/740857977816219678/1019885073610326077

![BC2F926E-AE8E-4BA1-8F54-3A853D0F7232](https://user-images.githubusercontent.com/59988195/190353264-2000abe1-60f5-43d4-8821-f48fe7ef8b91.jpeg)

**Comment by tersec** [comment]
https://github.com/status-im/nimbus-eth2/pull/4126 should fix this (shipped in v22.9.1)
